File: kata.py
import string
def alphabet_position(text):
    text = text.lower()
    special = [" ",".","'"]
    i=0
    result = ''
    alphabets = list(string.ascii_lowercase[:])
    while i <= len(text)-1: 
        character = text[i] #get the character by it index
        if character in special:#chek if its among the specal
            i+=1
        else:
            if character in alphabets:
                x = alphabets.index(character)+1
                result += " "+str(x)
                i+=1 
            else:
                i+=1              
    return  print(result.lstrip())
alphabet_position("The sunset sets at twelve o' clock. 9 ")

File: kata.rb
def alphabet_position(text)
    text = text.downcase
    alph = ('a'..'z').to_a()
    result=''
    counter = text.length()-1
    i = 0
    while i <= counter
        character  = text[i] 
        if  [".","'"," "].include? (character)
           i+=1 
        else   
            if alph.index(character) == nil
#               do nothing
            else
               result += ' '+(alph.index(character)+1).to_s
                i+=1
            end
        end
    end
    # result = result.join(" ")
    return  print result.lstrip 
end
alphabet_position("The sunset sets at twelve o' clock.")
